Tools/Scripts/filter-build-webkit

```diff
@@ -395,6 +405,11 @@ sub shouldIgnoreLine($$)
     return 1 if $line =~ /^.* Run script build phase '/;
     return 1 if $line =~ /^.* Traditional headermap style is no longer supported;/;
     return 1 if $line =~ /^.* Skipping duplicate build file in/;
+    # This is most likely due Xcode and the command-line compilation race to open the same .dia files.
```

Comment:
I don't think that's true. I see this error all the time in scenarios where Xcode isn't launched.

Reply:
added a new bug